### ISSUE TITLE
Add a section and SOP for transferring files between external collaborator

### ISSUE THREAD

**Issue by mathieuboudreau** [neuropoly/intranet.neuro.polymtl.ca#77]
Following up our conversation on slack about the large 16gb file transfer between us and an external collaborator for the #bavaria-quebec project.
